README.md

```diff
@@ -1 +1,7 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# Introduction of Yi Huang
+
+My name is Yi Huang, and I am originally from Shenzhen, China. I recently graduated from the University of Illinois at Urbana-Champaign, majoring in *psychology and economics*. **I had no experience in using R or Rmarkdown, but this homework is not as hard as I thought. The steps and instructions on the website are clear and straightforward.** My github homepage is https://github.com/yihuang97. 
```

Comment:
One way to make the link cleaner is to do this:
[My Github Homepage](your url)

That way the url is not displayed, but the text itself serves as the link.